challenges/challenge-1/challenge-1.html

```diff
@@ -279,15 +279,7 @@ <h1>Vital Julien</h1>
               Hickson
             </p>
           </div>
-          <!-- cell for TOC -->
-          <div class="camperCell col-md-5 col-lg-3" id="tatianatoc">
-            <h1>Tatiana</h1>
-            <img src="https://avatars0.githubusercontent.com/u/43054051?s=460&u=21998fb3fb9a6e124bede2920404dc359a0b63e0&v=4" alt="Tatiana">
-            <p>
-              "This too shall pass" - Virgin Mary
```

Comment:
@tacoelho I decided that I would like for you to change your quote. I believe it could be seen as making light of the Virgin Mary that is so dear to our Catholic neighbors. Here is our [Code of Conduct](https://github.com/DavidMatthewFraser/easy-to-git/blob/master/CODE_OF_CONDUCT.md).

Because I reverted a merged pull request, I think all of your html was reverted, so please include all the html you included.